packages/app/package.json

```diff
@@ -13,8 +13,7 @@
   },
   "main": "index.js",
   "scripts": {
-    "build": "ember build",
-    "prodbuild": "ember build --environment=production",
+    "build-ui": "cmd=\"ember build --environment=$npm_config_environment\" && echo $cmd && eval $cmd ",
```

Comment:
```suggestion
    "build": "ember build",
```

Reply:
unless we pass `--environment=$npm_config_environment` the variable doesn't get used.